src/lib.rs

```diff
@@ -150,12 +150,13 @@ pub async fn bootstap(mut shutdown: broadcast::Receiver<()>, config: Config) ->
             global_middleware.clone().layer(
                 CorsLayer::new()
                     .allow_methods([Method::GET, Method::POST, Method::DELETE])
-                    .allow_origin(
```

Comment:
Can you just update the terraform to * instead of a code change for that?

Reply:
That didn't work for me when I tried to test it in the functional tests.

Reply:
I think we'll need to do it slightly different.